package-res/resources/web/pentaho/data/_filter/_Element.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 define([
-  "pentaho/lang/Base"
+  "../../lang/Base"
```

Comment:
Ok. I think this can be further discussed, but as it is consistent with the current code, will merge.